### PR TITLE
Fix MetalLB YAML examples

### PR DESCRIPTION
**Pull request by cgoncalves**
Version(s): PR applies to all versions after a specific version: 4.11+

### CODE REVIEW

**Comment by cgoncalves** [comment]
@fedepaol @gkopels please review.

File: modules/nw-metallb-advertise-address-pool-with-bgp-advanced.adoc [hunk 1]
```diff
@@ -81,11 +81,11 @@ metadata:
   namespace: metallb-system
 spec:
   ipAddressPools:
-   - doc-example-bgp-adv
-  - communities:
+    - doc-example-bgp-adv
+  communities:
```
Comment (gkopels) [inline modules/nw-metallb-advertise-address-pool-with-bgp-advanced.adoc:85]: Maybe add here a different local pref such as 200

Comment (cgoncalves) [reply modules/nw-metallb-advertise-address-pool-with-bgp-advanced.adoc:85]: Why would adding such be relevant? It is just an example, it does not need to include all parameters.
The PR is about fixing syntax errors.

Comment (cgoncalves) [reply modules/nw-metallb-advertise-address-pool-with-bgp-advanced.adoc:85]: The "communities" change is a syntax error that I am fixing in this PR. Its intended value is left unchanged.

File: modules/nw-metallb-configure-specificpools-to-bgppeer.adoc [hunk 2]
```diff
@@ -130,13 +130,13 @@ metadata:
   namespace: metallb-system
 spec:
   ipAddressPools:
-  - pool1
+    - pool1
```
[the remark this answers is not in the document]
Comment (cgoncalves) [reply modules/nw-metallb-configure-specificpools-to-bgppeer.adoc:133]: Simply cosmetic change to align with the same indentation used in the examples on the same page.

File: modules/nw-metallb-configure-specificpools-to-bgppeer.adoc [hunk 3]
```diff
@@ -130,13 +130,13 @@ metadata:
...
   peers:
     - peer1
-  - communities:
-      - 65535:65282
- aggregationLength: 32
- aggregationLengthV6: 128
+  communities:
+    - 65535:65282
```
[the remark this answers is not in the document]
Comment (cgoncalves) [reply modules/nw-metallb-configure-specificpools-to-bgppeer.adoc:137]: Same as my previous comment: indentation consistency with other examples on the same page.
Both are syntactically valid and equal:
```
communities:
- 65535:65282
```

```
communities:
  - 65535:65282
```

**Comment by gkopels** [comment]
LGTM

**Comment by fedepaol** [comment]
/lgtm
Thanks!

**Comment by mburke5678** [comment]
@cgoncalves  PR needs a rebase to pick up changes to the Vale tests.

**Pull request merged**